model/system/metrics.yaml

```diff
@@ -32,6 +32,20 @@ groups:
     unit: "{cpu}"
     attributes: []
 
+  - id: metric.system.cpu.utilization
+    type: metric
+    metric_name: system.cpu.utilization
+    stability: development
+    brief: Fraction of total CPU time spent in active (non-idle) modes.
+    note: >
+      This metric provides a standardized and opinionated definition of CPU utilization.
+      It is defined as the proportion of CPU time spent in non-idle and non-iowait modes, following a formula similar to: 1 - (idle + iowait).
```

Comment:
Since this is a derived metric, could we become more descriptive here providing the exact formula that should be used for this calculation from the existing metrics?

I think it'd be extremely useful to show how we can calculate this from the existing `cpu.time` metrics for example.

Reply:
Good point, let me know what you think of  https://github.com/open-telemetry/semantic-conventions/pull/2088/commits/fc50a970aef5e44cef24f2fb82f26a69942ff84f